packages/gestalt/src/sharedSubcomponents/FormErrorMessage.tsx

```diff
@@ -31,6 +33,8 @@ export default function FormErrorMessage({
     mwebExperimentName: 'web_gestalt_visualRefresh',
   });
 
+  const dataTestIdIcon = dataTestId && `${dataTestId}-icon`;
```

Comment:
here we only look for the icon, what about the text?

The way this is implemented Icon is within Text. 

Can you breakdown for me how we wanna do error message, helper text, and max lenght?

Reply:
I'll add the anatomy for them in the documents, for this one, it is a little bit complex, as the {text} is not in a <Text> tag, I can add it to the upper <Box>, but it will include the <Icon> I think the best approach is to add <Text>, but it will impact current implementations (most likely on font change and integration tests)

Reply:
Change ready